dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/am/KubernetesApplicationManager.java

```diff
@@ -190,7 +190,7 @@ public String collectPodLog(KubernetesApplicationManagerContext kubernetesApplic
                 return logs;
             }
         } catch (Exception e) {
-            log.error("Collect pod log failed:", e.getMessage());
+            log.error("Collect pod log failed: {}", e.getMessage());
```

Comment:
Ideally this should be 👇 so the exception stack can be printed

```suggestion
            log.error("Collect pod log failed", e);
```

Reply:
Get it, thanks!